### ISSUE TITLE
Schema dump is not working as expected

### ISSUE THREAD

**Issue by pucinsk** [samesystem/graphql_rails#111]
The issue occured when I was trying to migrate from `graphql` and start using `graphql_rails` gem. 
`graphql` gem generates schema name from Rails app name with [`"#{Rails.application.class.parent_name}Schema"`](https://github.com/rmosolgo/graphql-ruby/blob/master/lib/generators/graphql/core.rb#L52). (Even though you can pass an option as a schema name but it is not mentioned in documentation for some reason)
And `graphql_rails` builds schema name from options passed to `schema:dump` task or takes an option and `[['graphql', name.presence, 'schema'].compact.join('_')`](https://github.com/samesystem/graphql_rails/blob/master/lib/graphql_rails/tasks/schema.rake#L23)
So when I try to dump a schema when using `graphql_rails` it doesn't find my schema file and throws an error
```
RET bundle exec rake graphql_rails:schema:dump machik
rake aborted!
NoMethodError: undefined method `to_definition' for nil:NilClass
/home/jokubas/work/bastardos/rails/graphql_rails/lib/graphql_rails/tasks/schema.rake:17:in `call'
/home/jokubas/work/bastardos/rails/graphql_rails/lib/graphql_rails/tasks/schema.rake:9:in `call'
```

We need to unify `graphql` and `graphql_rails` schema name generating

**Comment by povilasjurcys** [comment]
GraphqlRails is shifting to fixed schema file naming which is `graphql/graphql_schema.rb`. This is still experimental. Error happened because your schema name does is not `graphql/graphql_schema.rb`. Despite that it would be nice to have human friendly error raised

**Comment by povilasjurcys** [comment]
fixed by #130
